virt/release_notes/virt-4-20-release-notes.adoc

```diff
@@ -140,7 +141,7 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-20-ki-networking_{context}"]
 === Networking
 
-//CNV-38746: According to Petr, this will remain relevant as long as 4.12 is available. Anyone upgrading from 4.12 could be affected. Version 4.12 is supported until January 2026 
+//CNV-38746: According to Petr, this will remain relevant as long as 4.12 is available. Anyone upgrading from 4.12 could be affected. Version 4.12 is supported until January 2026
```

Comment:
This sentence feels to me like a leftover.
Can you please confirm?
@phoracek FYI

Reply:
This is commented out and not really part of the scope of this PR, it's only highlighted because of the removal of whitespace, so I would like to leave it as is for now in case a different writer is still using it.